app/client/cypress/integration/home/glossary.toggle.spec.js

```diff
@@ -0,0 +1,24 @@
+describe('Homepage', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it(`Glossary button toggles slide content`, () => {
+    cy.findAllByText('Glossary')
+      .filter(':visible')
+      .scrollIntoView()
+      .click();
```

Comment:
@julmcbride this should resolve the timing issue, and we won't need to resort to having Cypress scroll to elements (see my last comment for more info on why we probably don't want to do that here). The "data-disabled" attribute gets set to false after the Glossary constructor has done it's initialization, so yielding on that, and not just when it's visible is what we'll want to do.

```suggestion
    cy.findAllByText('Glossary')
      .filter('[data-disabled="false"]')
      .click();
```

Reply:
@courtneymyers Thanks, I tried this but test is failing: CypressError: Timed out retrying: Expected to find element: '[data-disabled="false"]', but never found it. Queried from element: [ <h2#glossary-title.sc-fzXfLS.kroACl>, 1 more... ]

I notice the Glossary button doesn't appear right away with the others (Data, About, etc.) It is a bit delayed in showing. I also get the "glossary is temporarily unavailable" message. I added GLOSSARY_AUTH to .env.local as advised by Andrew to resolve that before but it doesn't seem to be working now. I'll check with Andrew on that.